register.php

```diff
@@ -34,28 +34,31 @@
     }
 
     // Validate password
-    if (empty(trim($_POST["password"]))) {
```

Comment:
Need change in line 13

Reply:
![image](https://user-images.githubusercontent.com/32439965/75709440-ab777480-5c90-11ea-8acf-26b254563629.png)